app/src/main/java/ca/marshallasch/veil/PostListAdapter.java

```diff
@@ -190,6 +196,55 @@ public void sort(SortOption option) {
      */
     @Override
     public int getItemCount() {
-        return posts.size();
+        return postsFiltered.size();
+    }
+
+
+    @Override
+    public Filter getFilter()
+    {
+
+        /**
+         * This class is the tag filter. The CharSequence must be a list of tags that are
+         * denominated with a ':' character.
```

Comment:
Maybe use another character instead of ':' since its common to make a post like so "Sample Title: somthing somthing".

Reply:
This is for tags, not the title. The tag list is hard coded, we control whats in the tag list. I can switch it to a tab character tho because that should not ever be part of a valid tag.

Reply:
Right sorry, you can ignore this comment then.